clojure-mode.el

```diff
@@ -1886,6 +1886,19 @@ the cached value will be updated automatically."
 (defvar-local clojure-cached-ns nil
   "A buffer ns cache used to speed up ns-related operations.")
 
+(defun clojure--find-ns-in-direction (dir)
```

Comment:
Every time I see `dir` I think of directory, but that's not a big deal. It's a good idea to extract this.

Reply:
No reason to be lazy anyway. Changed it to direction.